app/script/ui/Shortcut.js

```diff
@@ -65,19 +65,6 @@ z.ui.Shortcut = (() => {
         },
       },
     },
-    [z.ui.ShortcutType.CALL_MUTE]: {
-      event: z.event.WebApp.SHORTCUT.CALL_MUTE,
-      shortcut: {
-        electron: {
-          macos: 'command + alt + m',
-          pc: 'ctrl + alt + m',
-        },
-        webapp: {
-          macos: 'command + alt + m',
-          pc: 'ctrl + alt + m',
-        },
-      },
-    },
```

Comment:
I think we should also remove the reject shortcut as the last remaining call realted shortcut.